src/server/index.js

```diff
@@ -520,6 +523,78 @@ module.exports = async function startServer(options) {
     ctx.body = { url, sceneId: scene_id };
   });
 
+  const nodePlatformToAssetPlatform = {
+    win32: "win",
+    darwin: "macos",
+    linux: "linux"
+  };
+  function getDownloadUrlForCurrentPlatform(assets) {
+    const assetPlatform = nodePlatformToAssetPlatform[process.platform];
+    return assets.find(asset => asset.name.includes(assetPlatform)).downloadUrl;
+  }
+
+  const updateInfoTimeout = 2000;
+
+  async function getLatestRelease() {
+    const token = "de8cbfb4cc0281c7b731c891df431016c29b0ace";
+    const result = await fetch("https://api.github.com/graphql", {
+      timeout: updateInfoTimeout,
+      method: "POST",
+      headers: { authorization: `bearer ${token}` },
+      body: JSON.stringify({
+        query: `
+          {
+            repository(owner: "mozillareality", name: "spoke") {
```

Comment:
do we need to include a order by here or something? I'm not that familiar with this API or graphql in general

Reply:
yeah looks like there is a way to order them, i'm guessing the default is probably OK but probably should confirm https://developer.github.com/v4/enum/releaseorderfield/

Reply:
it might be release date ascending by default, so we probably need to address this